apps/frontend/src/views/Login.vue

```diff
@@ -74,6 +74,9 @@ export default class Login extends Vue {
   signup() {
     this.$router.push('/signup');
   }
+  oauthLogin(site: string){
+    window.location.href = `/authn/${site}`;
+  }
```

Comment:
This doesn't seem to be used anywhere in this file. I see it is duplicated in LocalLogin.